specification/ai/Azure.AI.Projects/notifications/models.tsp

```diff
@@ -0,0 +1,104 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../common/models.tsp";
+import "../main.tsp";
+import "@typespec/openapi";
+import "@typespec/versioning";
+
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+
+namespace Azure.AI.Projects;
+
+@doc("Notification title information")
+@added(Versions.v2025_05_15_preview)
+model NotificationTitle {
+  @doc("The name of the notification")
+  name: string;
+
+  @doc("The description of the notification")
+  description: string;
+}
+
+@doc("View source action detail")
+@added(Versions.v2025_05_15_preview)
+model ViewSourceActionDetail {
+  @doc("The type of action")
+  type: string;
+
+  @doc("The target of the action")
+  target: string;
```

Comment:
how does the targeting work? is it based on an ID? some kind of expression language? something else? we should probably provide a more explicit name/documentation for this property.